gatorgrade/input/command_line_generator.py

```diff
@@ -0,0 +1,66 @@
+"""Generates a dictionary of shell and gator grader command options from a list of dict checks."""
+
+import os
+
+
+# Function to generate command options from a list of object checks
+
+# pylint: disable=too-many-nested-blocks
+def generate_checks(file_context_checks):
+    """Generate a dictionary of checks based on the configuration file.
+
+        This dictionary will have the format:
+        {
+            "shell": List of shell checks,
+            "gatorgrader": List of GatorGrader checks
+        },
+    Args:
+        file_context_checks: List containing dictionaries that contain file contexts
+            (either a file path or None if no file context)
+                and checks in another dictionary (can be either GatorGrader or shell checks).
+                    The input list is generated based on the configuration file.
```

Comment:
Your indentation here for the Args section of the docstring looks a little off. Would you be able to make the indentation uniform to improve readability?

Reply:
Done!